rotkehlchen/tests/unit/test_assets.py

```diff
@@ -90,7 +90,7 @@ def test_cryptocompare_asset_support(cryptocompare):
         'CTX',     # Centauri coin but CarTaxi in CC
         ethaddress_to_identifier('0xf14922001A2FB8541a433905437ae954419C2439'),  # noqa: E501 # Direct insurance token but DitCoin in CC
         'DRM',     # Dreamcoin but Dreamchain in CC
-        ethaddress_to_identifier('0x82fdedfB7635441aA5A92791D001fA7388da8025'),  # noqa: E501 # Digital Ticks but Data Exchange in CC
+        ethaddress_to_identifier('0x82fdedfB7635441aA5A92791D001fA7388da8025'),  # noqa: E501 # test_cryptocompare_asset_supportDigital Ticks but Data Exchange in CC
```

Comment:
What are you doing here? Is this a wrong copy paste?

Reply:
Yes, from a search and I didn't notice